tests/test_apertures_hsc.py

```diff
@@ -23,38 +23,54 @@
 
 import unittest
 import os
+import tempfile
 
+import lsst.daf.butler
 import lsst.utils
-import lsst.daf.persistence as dafPersist
 
 from lsst.fgcmcal.utilities import computeApertureRadiusFromDataRef, computeApertureRadiusFromName
 
+import fgcmcalTestBase
 
-class FgcmApertureTest(lsst.utils.tests.TestCase):
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+
+
+class FgcmApertureTestHSC(fgcmcalTestBase.FgcmcalTestBase, lsst.utils.tests.TestCase):
     @classmethod
     def setUpClass(cls):
         try:
             cls.dataDir = lsst.utils.getPackageDir('testdata_jointcal')
         except LookupError:
             raise unittest.SkipTest("testdata_jointcal not setup")
+        try:
+            lsst.utils.getPackageDir('obs_subaru')
+        except LookupError:
+            raise unittest.SkipTest("obs_subaru not setup")
 
-    def test_fgcmApertureHsc(self):
+        lsst.daf.butler.cli.cliLog.CliLog.initLog(longlog=False)
+
+        cls.testDir = tempfile.mkdtemp(dir=ROOT, prefix="TestFgcm-")
+
+        cls._importRepository('lsst.obs.subaru.HyperSuprimeCam',
+                              os.path.join(cls.dataDir, 'hsc/repo'),
+                              os.path.join(cls.dataDir, 'hsc', 'exports.yaml'))
+
+    def test_fgcmAperture(self):
         """
         Test computeApertureRadius for HSC.
         """
-        lsst.log.setLevel("HscMapper", lsst.log.FATAL)
-
-        butler = dafPersist.Butler(os.path.join(self.dataDir, 'hsc/repo'))
+        butler = lsst.daf.butler.Butler(self.repo, instrument='HSC', collections=['HSC/testdata'])
 
-        dataRef = butler.dataRef('src', visit=34648, ccd=51)
+        dataHandle = butler.getDeferred('src', visit=34648, detector=51)
```

Comment:
I see you’ve made a `dataRef` -> `dataHandle` name change here.  In my reading of the guidelines in section 4.2 of https://confluence.lsstcorp.org/display/DM/Gen2+Middleware+Removal+Planning, I’m not sure if this is formally _required_, but I did notice many residual `dataRef`-ish variable names in the main code which may warrant changing.  This would be a pretty onerous search-and-replace, so perhaps it’s worth getting @TallJimbo to weigh in on this?

Reply:
This is something that I wrestled with.  I know this is the correct style going forward, and so for new/updated code I was changing to the new nomenclature.  But I was also worried that if I edited too much on this ticket (which is primarily about removal), that it would make the review more challenging with more edits interspersed with deletions.  But I should either bite the bullet and make the edits here, or file a follow-up ticket to update the nomenclature.